src/main/resources/public/js/src/launches/logLevel/LogItemInfoTabs/LogItemInfoTabs.js

```diff
@@ -56,24 +56,40 @@ define(function (require) {
             'click [data-js-item-activity-label]': function () {
                 config.trackingDispatcher.trackEventNumber(203);
                 this.toggleModelField('activity');
+            },
+            'click [data-js-item-parametres-label]': function () {
+                this.toggleModelField('parametres');
             }
         },
 
         bindings: {
             '[data-js-item-stack-trace-label]': 'classes: {active: stackTrace}',
             '[data-js-item-gallery-label]': 'classes: {active: attachments}',
             '[data-js-item-details-label]': 'classes: {active: itemDetails}',
+            '[data-js-item-parametres-label]': 'classes: {active: parametres}',
             '[data-js-item-activity-label]': 'classes: {hide: isShowActivities, active: activity}',
             '[data-js-item-stack-trace]': 'classes: {hide: not(stackTrace)}',
             '[data-js-item-gallery]': 'classes: {hide: not(attachments)}',
             '[data-js-item-details]': 'classes: {hide: not(itemDetails)}',
-            '[data-js-item-activity]': 'classes: {hide: not(activity)}'
+            '[data-js-item-activity]': 'classes: {hide: not(activity)}',
+            '[data-js-item-parametres]': 'classes: {hide: not(parametres)}',
```

Comment:
Make a general class for the entire block, which will hide everything except the block with the notifications